x-pack/plugins/ingest_manager/server/services/epm/elasticsearch/transform/install.ts

```diff
@@ -24,11 +23,6 @@ interface TransformInstallation {
   content: string;
 }
 
-interface TransformPathDataset {
-  path: string;
-  dataset: Dataset;
-}
-
 export const installTransformForDataset = async (
```

Comment:
maybe this should be renamed to remove `ForDataset`?

Reply:
Yes will remove